rentals/rental.rb

```diff
@@ -11,4 +11,8 @@ def initialize(date, book, person)
     @person = person
     person.rentals << self
   end
+
```

Comment:
After you have created a `rental`, and want to see the list of rentals

Reply:
My guess is that you typed in a book or person number that's not listed in the 5th option (create a rental instance option) so the code stopped because it didn't account for numbers not in the list. No worries I've now child proofed the methods so that you can no longer insert numbers not in the list. 🙂